locales/nl/_excludes.json

```diff
@@ -0,0 +1,6 @@
+[
+    "Details",
+    "details",
```

Comment:
A key by that name does not exist.

Reply:
I added them based on https://laravel-lang.com/statuses-actions-nl.html#statuses-actions-nl-php where there is a `Details` in the json and a `details` in the php. Should the PHP ones be removed?

Reply:
It seems to have found another problem - keys are only checked against the JSON file avoiding checking against the PHP file.

I will fix this.